src/Buttons/Button.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, ReactElement } from "react";
```

Comment:
@Weetbix would be nice to test the new version with other repos using the button just to make sure it still works